src/hotspot/share/jfr/recorder/checkpoint/types/jfrTypeManager.cpp

```diff
@@ -218,6 +218,8 @@ bool JfrTypeManager::initialize() {
   register_type(TYPE_CODEBLOBTYPE, false, true, new CodeBlobTypeConstant());
   register_type(TYPE_VMOPERATIONTYPE, false, true, new VMOperationTypeConstant());
   register_type(TYPE_THREADSTATE, false, true, new ThreadStateConstant());
+  register_type(TYPE_BYTECODE, false, true, new BytecodeConstant());
```

Comment:
In the original PR these methods only have 3 arguments. The `require_safepoint` argument had been abstracted away by the time of the original PR. I had to add this argument in as the second argument. I set this to false like the other examples above.